### DISCUSSION TITLE
Web interface not always loading

### DISCUSSION THREAD

**Discussion by TownshendLights** [forkineye/ESPixelStick#582] [Q&A]
I'm running the 4.x dev dated sep7 2022 on ESP8266 D1 minis. 
Sometimes I can't access the board through the board's IP address unless I reboot. I'm using dedicated IP not DHCP.

Any ideas why please?

**Comment by MartinMueller2003** [comment]
Please keep the usb serial connected and post the output after it losses connectivity
